qrenderdoc/qrenderdoc.pro

```diff
@@ -137,7 +137,8 @@ win32 {
 		QMAKE_POST_LINK += $$_PRO_FILE_PWD_/../scripts/set_plist_version.sh $${RENDERDOC_VERSION}.0 $${INFO_PLIST_PATH}
 	} else {
 		QT += x11extras
-		DEFINES += RENDERDOC_PLATFORM_POSIX RENDERDOC_PLATFORM_LINUX RENDERDOC_WINDOWING_XLIB RENDERDOC_WINDOWING_XCB
+                DEFINES += RENDERDOC_PLATFORM_POSIX RENDERDOC_PLATFORM_LINUX RENDERDOC_WINDOWING_XLIB RENDERDOC_WINDOWING_XCB RENDERDOC_WINDOWING_WAYLAND
+        LIBS += -lwayland-client
```

Comment:
You'll need to add these defines/libs externally, at the moment this is being added unconditionally even if wayland support was not enabled. This is why the travis build failed until you apt-get'd libwayland-dev - the same thing would happen for an end-user if they didn't have libwayland-dev installed and we want to keep this optional.

If you look in `qrenderdoc/CMakeLists.txt` there's a `qrenderdoc_cmake.pri` include file that's included at the top of `qrenderdoc.pro` and is used to pass cmake configuration to qt. The way PySide2 is linked in only when found is a good example to copy.

Reply:
Cool, I'll make this change and revert the `linux_setup.sh` script.